Makefile

```diff
@@ -29,7 +29,11 @@ cyclo: ## Run gocyclo
 shellcheck: ## Run shellcheck
 	shellcheck **/*.sh
 
-style: fmt vet lint cyclo shellcheck ## Run all the formatting related commands (fmt, vet, lint, cyclo) + check shell scripts
+errcheck: ## Run errcheck
+	@echo "Running errcheck"
+	-./goerrcheck.sh
```

Comment:
What does the dash do?

Reply:
The dash tells `make` to ignore the exit value for that command. So, if `goerrcheck.sh` returns 1 (as in this case while we correct these problems), the "errcheck" rule won't fail

Reply:
Ok basically we have warning right now, ok

Reply:
I see, thank you